plugin/src/main/java/com/denizenscript/denizen/objects/PlayerTag.java

```diff
@@ -1242,9 +1242,18 @@ else if (ban.getExpiration() == null) {
                 return null;
             }
             ListTag list = new ListTag();
-            // TODO: optionally specify world
+            WorldTag world = null;
+            if (attribute.hasContext(1)) {
+                if (WorldTag.matches(attribute.getContext(1))) {

```

Comment:
Don't need to check matches, just error if there's context but the world value out of it is null

Reply:
okily